### ISSUE TITLE
Asked to submit this console error.

### ISSUE THREAD

**Issue by ADM1N1SH** [EssentialsX/Essentials#2345]
### Information

**Full output of `/ess version`**: *edit by @md678685: plugin not loading*

**Server log**: 
```java
[06:19:51] [Server thread/ERROR]: Exception when ADM1N1SH attempted to tab complete tpa at
org.bukkit.command.CommandException: Unhandled exception during tab completion for command '/tpa at' in plugin Essentials v2.16-pre4.9
	at org.bukkit.command.PluginCommand.tabComplete(PluginCommand.java:144) ~[spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at org.bukkit.command.Command.tabComplete(Command.java:90) ~[spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at org.bukkit.command.SimpleCommandMap.tabComplete(SimpleCommandMap.java:217) ~[spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at org.bukkit.craftbukkit.v1_13_R1.CraftServer.tabCompleteCommand(CraftServer.java:1641) ~[spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at org.bukkit.craftbukkit.v1_13_R1.CraftServer.tabComplete(CraftServer.java:1613) ~[spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at org.bukkit.craftbukkit.v1_13_R1.command.BukkitCommandWrapper.getSuggestions(BukkitCommandWrapper.java:49) ~[spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at com.mojang.brigadier.tree.ArgumentCommandNode.listSuggestions(ArgumentCommandNode.java:68) ~[spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at com.mojang.brigadier.CommandDispatcher.getCompletionSuggestions(CommandDispatcher.java:372) ~[spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at net.minecraft.server.v1_13_R1.PlayerConnection.a(PlayerConnection.java:503) ~[spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at net.minecraft.server.v1_13_R1.PacketPlayInTabComplete.a(SourceFile:36) ~[spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at net.minecraft.server.v1_13_R1.PacketPlayInTabComplete.a(SourceFile:9) ~[spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at net.minecraft.server.v1_13_R1.PlayerConnectionUtils.a(SourceFile:10) ~[spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at net.minecraft.server.v1_13_R1.PlayerConnectionUtils$$Lambda$1839.00000000A006A990.run(Unknown Source) ~[?:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_192]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_192]
	at net.minecraft.server.v1_13_R1.SystemUtils.a(SourceFile:198) [spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at net.minecraft.server.v1_13_R1.MinecraftServer.w(MinecraftServer.java:884) [spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at net.minecraft.server.v1_13_R1.DedicatedServer.w(DedicatedServer.java:411) [spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at net.minecraft.server.v1_13_R1.MinecraftServer.v(MinecraftServer.java:819) [spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at net.minecraft.server.v1_13_R1.MinecraftServer.run(MinecraftServer.java:717) [spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	at java.lang.Thread.run(Thread.java:813) [?:1.8.0_192]
Caused by: java.lang.NullPointerException
	at com.earth2me.essentials.Essentials.onTabCompleteEssentials(Essentials.java:378) ~[?:?]
	at com.earth2me.essentials.Essentials.onTabComplete(Essentials.java:370) ~[?:?]
	at org.bukkit.command.PluginCommand.tabComplete(PluginCommand.java:135) ~[spigotmc-1.13-R0.1.jar:git-Spigot-fe3ab0d-162bda9]
	... 20 more
```

**EssentialsX config**: I have not touched the config since downloading it. It is the standard config for this version.

### Details

**Description**  
It's an error message...

**Steps to reproduce**  
Downloaded essentials release per advice given to other people. Restarted server after installation. This happened when trying to use the commands.

**Expected behavior**  
 I expected to be able to use essential permissions/commands.

**Screenshots**

**Comment by ADM1N1SH** [comment]
None of the other stuff is showing despite filling it out. Was I supposed to delete stuff?

**Comment by mdcfe** [comment]
Please run `/ess version` and give us the full output of the command, as requested in the template.

**Comment by mdcfe** [comment]
My apologies, I wasn't paying attention (the version command doesn't work if EssentialsX doesn't load). Could you get the output of `/version`?

**Comment by ADM1N1SH** [comment]
19:53:36] [Server thread/INFO]: This server is running CraftBukkit version git-Spigot-642f6d2-57ab4cf (MC: 1.12.2) (Implementing API version 1.12.2-R0.1-SNAPSHOT)[m

Thank you for explaining that to me <3

**Comment by mdcfe** [comment]
I believe this was the issue in #2346 - could you update to the [latest EssentialsX release](https://ci.ender.zone/job/EssentialsX) and see if it still occurs?

**Comment by ADM1N1SH** [comment]
I guess I'm confused here, in the comments where I saw the answer to use this, the version I downloaded was the developer build for 1.13. 

So you are telling me to backdate the plugin, which will put me into the problem I had using the latest release in the first place?

I know releases and responses can be a bit mismatched based on dates, so I might just be entirely confused. I'll give it a try and let you know here in a bit.

**Comment by mdcfe** [comment]
No, we've officially released EssentialsX 2.16 now, and so the builds I linked above are more up-to-date than the experimental builds (which have been removed).

I notice that your error log is from a 1.13.x server, but your `/version` command seems to be from a 1.12.2 server - could you clarify what server you're running?

**Comment by triagonal** [comment]
Closed due to lack of response.
